Update README in develop to reflect 7.4 release (#709)
rticommunity/rticonnextdds-examples: README.md

## Versioning

> The examples contained in this branch have been built and tested against **RTI Connext

Reviewer: ```suggestion
The examples contained in this branch were built and tested against **RTI Connext
```

7.4.0 EAR**. If you need examples that have been built and tested against older

> versions of RTI Connext DDS, please check out **master** or the appropriate branch:

Reviewer: ```suggestion
versions of RTI Connext, please check out **master** or the appropriate branch:
```
